.git-hooks/locale.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+/**
+ * What is this? With this small JavaScript file we want 
+ * to block the upload of modified files in the "locale" 
+ * folder to the repository through a git hook generated 
+ * with the Husky library.
+ * 
+ * Usage: $ node .git-hook/locale.js
+ * 
+ * Happy releasing!
+ */
+
+'use strict'
+
+// Require main dependences.
+const { execSync } = require('child_process')
+const fs = require('fs')
+const path = require('path')
+
+// Execute Git Diff for get all files with changes.
+const results = execSync('git diff --cached --name-only')
+
+// Create array with all files.
+const arrayResults = results.toString().split('\n')
+// Remove last element with empty line.
+arrayResults.pop()
+
+// Define array files to not change.
+const filesNoChange = []
+
+function gitHook(expresion) {
+
+  // Define RegExp with reg param.
+  const reg = new RegExp(expresion, 'g')
+
+  // Iterate in array for search if RegExp is true or not.
+  arrayResults.forEach(file => {
+    // Control for search if exist or not file.
+    if (fs.existsSync(path.join(__dirname, '..', file)) && reg.test(path.join(__dirname, '..', file))) {
+      // Include path reference into array for resolve result console.
+      filesNoChange.push({
+        Path: path.join(__dirname, '..', file)
+      })
+    }
+  })
+
+  // Show an error and log table with the list of modified files in the "locale" folder.
+  if (filesNoChange.length !== 0) {
+    console.error('\x1b[31m', '\n ❌ You can not commit changes of the following files:', '\x1b[0m')
```

Comment:
What is `x1b[31m`? Can we live without it?

Reply:
We can live without this code :). I would like to keep it, add the red color to the warning text.

Reply:
![image](https://user-images.githubusercontent.com/28801003/82358611-6bae6680-9a0f-11ea-88bc-c540e7c6ca29.png)

For me it looks okay with red cross and normal text, so I think we can live without it.

Reply:
Done with commit of @alexandrtovmach